File: Hello_Python/day03/ooptest.py
class Animal:
    def __init__(self):
        print("Animal:__init__")
        self.age = 1
    def getOld(self):
        self.age += 1
    def __del__(self):
        print("Animal:__del__")
    
class Cat(Animal):
    def __init__(self):
        print("Cat:__init__")
        super().__init__()
        self.ssagaji = False
    def chur(self):
        self.ssagaji = True
    def __del__(self):
        print("Cat:__del__")
        

if __name__=='__main__':
    
    ani = Animal()
    print(ani.age)
    ani.getOld()
    print(ani.age)
    
    cat = Cat()
    print(cat.age)
    print(cat.ssagaji)
    cat.getOld()
    cat.chur()
    print(cat.age)
    print(cat.ssagaji)

File: DDIT_Python/Hello_Java/src/day02/OopTest.java
package day02;

public class OopTest {
	public static void main(String[] args) {
		Animal ani = new Animal();
		System.out.println(ani.age);
		ani.getOld();
		
		Cat cat = new Cat();
		System.out.println(cat.age);
		System.out.println(cat.ssagaji);
		cat.getOld();
		cat.chur();
		System.out.println(cat.age);
		System.out.println(cat.ssagaji);
		
	}
}
